utilities/src/main/java/io/syndesis/qe/utils/GoogleSheetsUtils.java

```diff
@@ -0,0 +1,111 @@
+package io.syndesis.qe.utils;
+
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.model.ClearValuesRequest;
+import com.google.api.services.sheets.v4.model.DeleteSheetRequest;
+import com.google.api.services.sheets.v4.model.Request;
+import com.google.api.services.sheets.v4.model.BatchUpdateSpreadsheetRequest;
+import com.google.api.services.sheets.v4.model.Sheet;
+import com.google.api.services.sheets.v4.model.Spreadsheet;
+import com.google.api.services.sheets.v4.model.ValueRange;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+public class GoogleSheetsUtils {
+    @Autowired
+    private GoogleAccounts accounts;
+    private Sheets sheets;
+    public static String TEST_SHEET_ID = "";
+    public static String DATA_TEST_SPREADSHEET = "1_OLTcj_y8NwST9KHhg8etB10xr6t3TrzaFXwW2dhpXw";
```

Comment:
no need for these to be static, since you have instance of `GoogleSheetsUtils` everywhere

Reply:
@mmelko you missed these 8 comments ^